pagopa/io-web-profile
# [IOPID-719] : Magic link page

## Short description

Created magic link page

## List of changes proposed in this pull request

- Magic link page
- Renamed expired link page according to confluence doc: https://pagopa.atlassian.net/wiki/spaces/IAEI/pages/761528580/IO-ESCO+URL+Portale
- Fixed double vars inside env file

## How to test

yarn run build
check that in the folder **out/it/blocco-accesso/** exists **magic-link** and **link-scaduto** pages
-yarn run start-static
go to **/it/blocco-accesso/magic-link** & **/it/blocco-accesso/link-scaduto**
